firenoc/firenoc-calculator/migration/migrate.sh

```
@@ -1,2 +1,2 @@
 #!/bin/bash
-flyway -url=$DB_URL -table=$SCHEMA_TABLE -user=$FLYWAY_USER -password=$FLYWAY_PASSWORD -locations=$FLYWAY_LOCATIONS -baselineOnMigrate=true -outOfOrder=true  migrate
\ No newline at end of file
+flyway -url=$DB_URL -table=$SCHEMA_TABLE -user=$FLYWAY_USER -password=$FLYWAY_PASSWORD -locations=$FLYWAY_LOCATIONS -baselineOnMigrate=true -outOfOrder=true migrate
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)